config/default.yml

```diff
@@ -3356,12 +3356,16 @@ Style/ImplicitRuntimeError:
   VersionAdded: '0.41'
 
 Style/InfiniteLoop:
-  Description: 'Use Kernel#loop for infinite loops.'
+  Description: >-
+                 Use Kernel#loop for infinite loops.
+                 This cop is marked as unsafe as if the rule does not necessarily
```

Comment:
Same here. I generally try to keep those description short, so we don't end up duplicating too much. At some point we'll probably have to start auto-generating them from the cop classes, but that's not a big deal.

Reply:
Shortened!